projects/aws-containers/hello-eks-anywhere/conf/default.conf

```diff
@@ -0,0 +1,24 @@
+map $http_user_agent $root {
```

Comment:
Did we need to change what is in the source repo? https://github.com/aws-containers/hello-eks-anywhere/tree/main/conf